botbot/botparser.py

```diff
@@ -17,7 +17,8 @@
 EVAL_FUNCTIONS = DEFAULT_FUNCTIONS.copy()
 EVAL_FUNCTIONS.update({'bool': bool, 'repr': repr, 'to_json': lambda x: json.dumps(x), 'from_json': lambda x: json.loads(x), 'len': len, 'mention': euphutils.mention, 'unwhitespace': lambda x: re.sub(r'\s', '_', x), 'time': time.time, 'eval': None})
 EVAL_OPERATORS = DEFAULT_OPERATORS.copy()
-EVAL_OPERATORS.update({ast.Not: operator.not_, ast.In: lambda a, b: a in b, ast.NotIn: lambda a, b: a not in b, ast.Is: operator.is_, ast.IsNot: operator.is_not})
+# XXX: AFAICT repr() is safe for our purposes.
+DISALLOW_FUNCTIONS.discard(repr)
```

Comment:
I can't imagine `repr()` being unsafe for any built-in Python class, but I'm also okay with disallowing it for now unless someone has a use case.